### PR TITLE
Added edX styling to Bootstrap

### PR DESCRIPTION
**Pull request by clintonb**
The default, primary, and link buttons have been updated according to the edX pattern library. A demo page is included to demonstrate the new styling.

XCOM-367

@frrrances @jimabramson @rlucioni

### CODE REVIEW

File: ecommerce/urls.py [hunk 1]
```diff
@@ -70,6 +70,8 @@ def handler403(_):
         url(r'^403/$', handler403, name='403'),
         url(r'^404/$', 'django.views.defaults.page_not_found', name='404'),
         url(r'^500/$', 'django.views.defaults.server_error', name='500'),
+        url(r'^bootstrap/$', TemplateView.as_view(template_name='bootstrap-demo.html')),
```
Comment (jimabramson) [inline ecommerce/urls.py:82]: would it be worth only adding this when DEBUG is True ?

File: ecommerce/static/sass/_base.scss [hunk 2]
```diff
@@ -1,3 +1,56 @@
 // app styling: base
 // ====================
 // NOTE: used for any basic styling or overriding that isn't covered by Bootstrap or theming
+body {
```
Comment (rlucioni) [inline ecommerce/static/sass/_base.scss:4]: Is this the best way to use the pattern library? What if something in the pattern library changes? We'd need to update it here, separately.

Comment (clintonb) [inline ecommerce/static/sass/_base.scss:4]: For now, yes. As we discussed in grooming, there is no perfect integration of the edX pattern library with Bootstrap.

File: ecommerce/urls.py (hunk 1 above)
Comment (clintonb) [inline ecommerce/urls.py:82]: It is. See line 66 (which Github is hiding).

File: ecommerce/static/sass/_base.scss (hunk 2 above)
Comment (rlucioni) [inline ecommerce/static/sass/_base.scss:4]: Thanks; I hadn't seen this before grooming.

**Comment by rlucioni** [comment]
:+1:

**Pull request merged**